charts/rancher-vsphere-cpi/Chart.yaml

```diff
@@ -5,10 +5,10 @@ annotations:
   catalog.cattle.io/namespace: kube-system
   catalog.cattle.io/os: linux
   catalog.cattle.io/permits-os: linux,windows
-  catalog.cattle.io/rancher-version: '>= 2.6.0-0'
+  catalog.cattle.io/rancher-version: '>= 2.8.0-0'
   catalog.cattle.io/release-name: vsphere-cpi
 apiVersion: v1
-appVersion: 1.26.1
+appVersion: 1.27.0
```

Comment:
upstream just released support for 1.28.0, which we'll want for RKE2. Any chance you could roll those in to this PR?

https://github.com/kubernetes/cloud-provider-vsphere/releases/tag/v1.28.0

Reply:
Absolutely, I'm on it.

Reply:
This issue is specifically for 1.27 support, I guess we could.. if we don't separate the bumps to k8s 1.27 and 1.28 though and there are regressions/issues with deploying the chart we'll have to debug which upgrade to which version is causing the problem. I'd rather track it separately. Let me check with the team

Reply:
I would rather keep this one 1.27 specific and do another follow-up PR with 1.28 versions, probably after 1.28-supporting CSI chart is available as well.